sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -62,9 +62,10 @@ public final class StorageBlobClient: PipelineClient {
 
     fileprivate var managing = false
     fileprivate lazy var manager: TransferManager = {
-        let instance = URLSessionTransferManager(delegate: self, logger: self.logger)
-        instance.loadContext()
-        return instance
+        var manager = URLSessionTransferManager.shared
```

Comment:
Re-verified that creating multiple Transfer Managers (really creating multiple ManagedObjectContexts) causes unworkable problems with CoreData. You will reliably, but unpredictably get CoreData failures because "blocks was invalid" due to it trying to save in different contexts.

Reply:
I guess I'm not surprised. Using a static shared instance shouldn't be too much of a problem, but there's a bug here that now each newly-initialized Client will overwrite the `delegate` and `logger` properties of the shared TransferManager instance. We'll need to have some way of setting those properties on each new Transfer instead of on the TransferManager itself.

Reply:
I guess I'm not surprised. Using a static shared instance shouldn't be too much of a problem, but there's a bug here that now each newly-initialized Client will overwrite the delegate and logger properties of the shared TransferManager instance. We'll need to have some way of setting those properties on each new Transfer instead of on the TransferManager itself.

Reply:
Right, ideally I'd like to just have the PersistentContainer be shared (which ultimately yields the ManagedObjectContext), but I think a separate PR would be appropriate, since that would be a non-trivial change.

Reply:
That's fine, but we have a known bug here so let's at least mark it as a TODO.